setup.cfg

```diff
@@ -1,3 +1,3 @@
 [metadata]
-description-file=README.md
+description_file=README.md
```

Comment:
This is fixed in another PR.

Reply:
I had to do it since this branch is being used by a customer in a colab.